packages/react-components/react-virtualizer/src/components/VirtualizerScrollViewDynamic/VirtualizerScrollViewDynamic.ts

```diff
@@ -0,0 +1,35 @@
+import { VirtualizerScrollViewDynamicProps } from './VirtualizerScrollViewDynamic.types';
+import { useVirtualizerScrollViewDynamic_unstable } from './useVirtualizerScrollViewDynamic';
+import {
+  renderIVirtualizerScrollViewDynamic_unstable,
+  renderVirtualizerScrollViewDynamic_unstable,
+} from './renderVirtualizerScrollViewDynamic';
+import { useVirtualizerScrollViewDynamicStyles_unstable } from './useVirtualizerScrollViewDynamicStyles';
+import * as React from 'react';
+import { useVirtualizerContextState, VirtualizerContextProps } from '../../Utilities';
+
+/**
+ * Virtualizer ScrollView
+ */
+
+export const IVirtualizerScrollViewDynamic: React.FC<VirtualizerScrollViewDynamicProps> = (
```

Comment:
I might need a better naming system here - this is the 'internal' class that is required to be wrapped in a local context. We use this method as the base virtualizer hooks also need to expose external methods for accessing the current index prior to a render, as well as a chance for users to inject custom index logic should it be needed.

Reply:
Updated to 'ContextlessVirtualizerScrollViewDynamic' so that users don't accidentally use/discover while typing 'Virtualizer*'